test/uk/gov/hmrc/helptosavefrontend/views/register/ViewBehavioursSpec.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavefrontend.views.register
+
+import org.jsoup.nodes.Document
+import play.api.i18n.MessagesApi
+import play.api.test.FakeRequest
+import uk.gov.hmrc.helptosavefrontend.TestSupport
+
+class ViewBehavioursSpec extends TestSupport {
+
+  lazy val injector = fakeApplication.injector
+  lazy val request = FakeRequest()
+  def messagesApi = injector.instanceOf[MessagesApi]
```

Comment:
this one could be a lazy val - doesn't need to be evaluated every time it's called